cmake/Tools/Platform/Android/android_post_build.py

```diff
@@ -0,0 +1,256 @@
+    :param tgt:             The target folder to synchronize to
+    """
+
+    assert not tgt.is_file()
+    assert src.is_dir()
+
+    # Make sure the target folder exists
+    tgt.mkdir(parents=True, exist_ok=True)
+
+    # Iterate through the items in the source folder and copy to the target folder
+    processed = 0
+    for src_item in src.iterdir():
+        tgt_item = tgt / src_item.name
+        copy_or_create_link(src_item, tgt_item)
+        processed += 1
+
+    logger.info(f"{processed} items from {src} linked/copied to {tgt}")
+
+
+def apply_pak_layout(project_root: Path, asset_bundle_folder: str, target_layout_root: Path) -> None:
+    """
+    Apply the pak folder layout to the target assets folder
+
+    :param project_root:            The project root folder to base the search for the location of the pak files (Bundle)
+    :param asset_bundle_folder:     The sub path within the project root folder of the location of the pak files
+    :param target_layout_root:      The target layout destination of the pak files
+    """
+
+    src_pak_file_full_path = project_root / asset_bundle_folder
+
+    # Make sure that the source bundle folder where we look up the paks exist
+    if not src_pak_file_full_path.is_dir():
+        raise AndroidPostBuildError(f"Pak files are expected at location {src_pak_file_full_path}, but the folder doesnt exist. {PAK_FILE_INSTRUCTIONS}")
+
+    # Make sure that we have at least the engine_android.pak file
+    has_engine_android_pak = False
+    for pak_dir_item in src_pak_file_full_path.iterdir():
+        if pak_dir_item.is_file and str(pak_dir_item.name).lower() == f'engine_{ASSET_PLATFORM_KEY}.pak':
+            has_engine_android_pak = True
+            break
+    if not has_engine_android_pak:
+        raise AndroidPostBuildError(f"Unable to located the required 'engine_android.pak' file at location specified at {src_pak_file_full_path}. "
+                                    f"{PAK_FILE_INSTRUCTIONS}")
+
+    # Clear out the target folder first
+    safe_clear_folder(target_layout_root)
+
+    # Copy/Link the contents to the target folder
+    synchronize_folders(src_pak_file_full_path, target_layout_root)
+
+
+def apply_loose_layout(project_root: Path, target_layout_root: Path, android_app_root_path: Path, native_build_folder: str, build_config: str) -> None:
+    """
+    Apply the loose assets folder layout rules to the target assets folder
+
+    :param project_root:            The project folder root to look for the loose assets
+    :param target_layout_root:      The target layout destination of the loose assets
+    :param android_app_root_path:   The root of the 'app' project in the Gradle script
+    :param native_build_folder:     The sub-folder where the intermediate native project files are generated
+    :param build_config:            The build configuration used for the native build
+    :param is_monolithic:           Flag marking the deployment is for monolithic builds
```

Comment:
`is_monolithic` is no longer a parameter
```suggestion
```

Reply:
`android_app_root_path`, `native_build_folder`, and `build_config` aren't used anymore either.